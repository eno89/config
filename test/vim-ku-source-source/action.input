" Test: vim-ku-source-source action
runtime! plugin/ku.vim plugin/ku/source/source.vim
call vspec#hint({'scope': 'ku#_local_variables()', 'sid': 'ku#_sid_prefix()'})
function s:describe__ku_source_source_action_argdelete()  "{{{1
  It requires the following initialization

  Should Call('s:ku_active_p') == 0

  It should succeed

  let source_name_a = 'source'

  Should ku#source#source#action_open({'word': source_name_a}) is 0

  Should Call('s:ku_active_p') == 1
  Should Ref('s:session').sources ==# [ku#available_sources()[source_name_a]]

  It requires to reset side effect

  silent! call ku#start()
  Should Call('s:ku_active_p') == 1
  bwipeout!
  ResetContext
endfunction




" __END__  "{{{1
" vim: filetype=vim foldmethod=marker
