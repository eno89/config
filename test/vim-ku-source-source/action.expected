==== ku_source_source_action_argdelete 
---- It requires the following initialization 
.
---- It should succeed 
...
---- It requires to reset side effect 
.



**** Result ****


5 examples, 0 failures
